src/mongoLocker.ts

```diff
@@ -0,0 +1,14 @@
+import getMongoLocker, { LockKey, LockCallback, LockerOptions } from './getMongoLocker';
+import getCollection from "./getCollection";
+
+let lockerPromise: ReturnType<typeof getMongoLocker>;
+
+export default async function <T>(key: LockKey, callback: LockCallback<T>, options: LockerOptions = {}): Promise<T> {
+    if (!lockerPromise) {
+        lockerPromise = getMongoLocker(getCollection('appLocks'));
```

Comment:
Good idea. It would be good to add also some note in readme that from now on - you should use mongoLocker instead of getMongoLocker and that you should prefix your keys (instead of using different collections) to avoid conflicts.

Reply:
added to readme